libs/natparse/src/main/java/org/amshove/natparse/parsing/ParserErrors.java

```diff
@@ -224,7 +224,7 @@ public static IDiagnostic optionalNotAllowedInCurrentScope(TokenNode errorToken,
 	public static IDiagnostic trailingToken(SyntaxToken token)
 	{
 		return ParserDiagnostic.create(
-			"Trailing token <%s> not allowed here".formatted(token.kind()),
+			"Trailing token <%s> not allowed here, line number <%s>".formatted(token.kind(), token.line()),
```

Comment:
The line number is redundant and can be removed, as it is part of the diagnostic.

The called method handles the line number and does special handling if the token is from a copycode that is included.